package.json

```diff
@@ -78,6 +82,8 @@
     "transform": {
       "\\.(ts)$": "ts-jest"
     },
+    "testEnvironment": "node",
```

Comment:
Jest normally runs in "jsdom", which is a browser like environment. Since this is a node app, we should run the tests in "node".
https://jestjs.io/docs/en/configuration